packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.test.tsx

```diff
@@ -14,6 +14,9 @@ jest.mock('@elastic/eui', () => ({
   useIsWithinBreakpoints: (args: string[]) => {
     return args[0] === 'xs';
   },
+  EuiSideNav: function Component() {
+    // no-op
```

Comment:
Shouldn't a functional component always return? It seems that for consistency we should return `null`. WDYT?

Reply:
I think when returning null we'd explicitly want it to be null. By "no-op" we're saying "I don't care what this returns".